web/src/components/FOI/Home/Redlining.js

```diff
@@ -2690,6 +2768,18 @@ const Redlining = React.forwardRef(
               documentsObjArr = [];
             }
           }
+          let annotList = annotationManager.getAnnotationsList();

```

Comment:
This finds the annotations with label s. 14 and matches them to the redaction annotations by id. It makes the following assumptions about the Annotation object:
Annotation.hR is where the "s. 14" label is stored
Annotation.Aj.parentRedaction is where the id of the parent redaction is stored
Annotation.Zk is where the id of the redaction annotation is stored